assets/css/main.css

```diff
@@ -153,31 +153,22 @@ Content
 }
 
 .tabs {
-	@apply flex items-stretch font-head p-0 list-none flex-col md:flex-row border-b border-card-dark;
+	@apply flex items-stretch font-head p-0 list-none flex-col md:flex-row border-b border-card-light;
 }
 
 .tab-jobs {
 	@apply items-center block md:flex md:flex-wrap md:flex-grow w-full lg:w-1/2 md:justify-evenly font-head mt-4 lg:mt-0;
 }
 
-.tab > label {
-	@apply block md:hidden px-3 py-3 cursor-pointer;
-	margin-bottom: -1px;
-	font-size: 1.5rem; 
-	font-weight: 600;
-	text-transform: uppercase;
-	letter-spacing: 1px;
-	transition: all 0.3s;
-	background: #1E1E1F;
-	border-right: 2px solid #141415;
-}
-
```

Comment:
There's duplicate styling below

Reply:
Thanks for the explanation ahead of time by the way.